services/galley/src/Galley/API.hs

```diff
@@ -318,6 +331,8 @@ sitemap = do
       .&> zauthBotId
       .&. zauthConvId
       .&. accept "application" "json"
+  -- This endpoint can lead to the following events being sent to clients:
```

Comment:
What does clients mean here?

Reply:
Similar to above comment, client devices of the recipients (users).